packages/main/src/TabContainerPopover.hbs

```diff
@@ -12,7 +12,11 @@
 			{{#each _endOverflowItems}}
 				{{this.overflowPresentation}}
 			{{/each}}
-		{{else}}
+		{{else if _tabItems.length}}
+			{{#each _tabItems}}
```

Comment:
Can you refactor this popover to only iterate over one array? That way, TabContainer will maintain only one array of tabs to show at a time.